include/desktop/output.h

```diff
@@ -25,6 +25,11 @@ struct kiwmi_output {
 
     int damaged;
 
+    /* Used in output.c:output_manager_configure(), to keep track of which
+	 * outputs have been enabled or disabled.  Simpler to put it here, than
+	 * to create a separate, function local data structure. */
+    bool enable_changed;
```

Comment:
Perhaps we don’t need this in the end, coz there’s `wlr_output_state.committed` and `WLR_OUTPUT_STATE_ENABLED`. I’ll check yet if we can use it or not.